connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/application/ReplE2ESuite.scala

```diff
@@ -51,29 +50,26 @@ class ReplE2ESuite extends RemoteSparkSession with BeforeAndAfterEach {
   }
 
   override def beforeAll(): Unit = {
-    // TODO(SPARK-44121) Remove this check condition
```

Comment:
I suggest these changes could be merged into 3.5 to avoid future conflicts. WDYT @dongjoon-hyun

Reply:
At least the current one related to connect.

Reply:
For this area, I'll follow your guideline because you are the original author, @LuciferYang . :)